src/test/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvertTest.java

```diff
@@ -83,34 +60,23 @@ void apartmentsToJsonTest() throws IOException, URISyntaxException {
 
 		String expectedApartment = "[{\"address\":\"118 rue du père noel 77480\",\"description\":\"\",\"floorArea\":1182118.48,\"floorAreaTerrace\":0.0,\"nbBathrooms\":0,\"nbBedrooms\":0,\"nbMinNight\":0,\"nbSleeping\":0,\"pricePerNight\":0.0,\"tele\":false,\"terrace\":false,\"title\":\"Grand Igloo\",\"wifi\":false},{\"address\":\"123 rue du soleil\",\"description\":\"\",\"floorArea\":1234567.89,\"floorAreaTerrace\":0.0,\"nbBathrooms\":0,\"nbBedrooms\":0,\"nbMinNight\":0,\"nbSleeping\":0,\"pricePerNight\":0.0,\"tele\":false,\"terrace\":false,\"title\":\"Maison Test\",\"wifi\":false}]";
 
-		URI ressource = JsonConvertTest.class.getResource("jsonListTest.json").toURI();
+		URI ressource = JsonConvertTest.class.getResource("jsonApartments.json").toURI();
 		Path jsonPath = Path.of(ressource);
 
 		JsonConvert.apartmentsToJson(apartments, jsonPath);
```

Comment:
Ne pas écrire dans votre class path. Ajoutez une méthode qui écrit dans un String, et utilisez-la pour tester.

Reply:
Fix in commit 79a20a634d95f956f81cdae25561e0526b354f54